dbt_platform_helper/utils/validation.py

```diff
@@ -677,13 +679,35 @@ def _validate_environment_pipelines_triggers(config):
         abort_with_error(error_message + "\n  ".join(errors))
 
 
+def lint_yaml_for_duplicate_keys(file_path):
+
+    lint_yaml_config = """
+rules:
+  key-duplicates: enable
```

Comment:
Are there any additional rules we could make use of?

Reply:
I think that my be scope creep, and increase the complexity of this pull requests, how the error is handled, outputted etc.

Reply:
Fair